src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Applying common sense: I would just keep the Mono "copyright" there. Nothing prevents us to extend Mono files and modify them. Even upon initial checkin AFAIK. The key thing is to acknowledge where it came from. Of course, I am not a lawyer, maybe we should ask ...